src/utils/countries-api.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable consistent-return */
+import axios from 'axios';
+
+export const transferDate = (date) => date.toISOString().split('T')[0];
+
+const today = new Date();
+let todayDate = transferDate(today);
+
+const yesterday = new Date();
+yesterday.setDate(today.getDate() - 1);
+const yesterDate = transferDate(yesterday);
+
+const europecountries = ['spain', 'germany', 'italy', 'france', 'denmark', 'belgium', 'portugal', 'netherlands', 'switzerland', 'united_kingdom'];
+
+const config = {
+  method: 'get',
+  url: `https://api.covid19tracking.narrativa.com/api/${todayDate}`,
+};
+
+const fetchCountry = async () => {
+  const res = await axios(config);
+  if (res.status === 200) {
+    const contriesData = await res.data.dates[`${todayDate}`].countries;
+    const allCountries = Object.keys(contriesData).map((key) => contriesData[key]);
+    const filteredCountries = allCountries.filter((obj) => europecountries.includes(obj.id));
+
+    // Non camelCase are needed here since they're used by the Narrativa API.
+
+    /* eslint-disable camelcase */
```

Comment:
- [ ] please kindly avoid using several eslint-disable comments to get around ESlint errors. these ESlint errors are at best showing that you are not adhering to some coding standards or at worst highlighting a possible underlying issue with your code. please kindly address these issues.

Reply:
That's just the best I can do for now. I've tried to correct it.

Reply:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean as recommended.